src/core_plugins/kibana/public/discover/controllers/discover.js

```diff
@@ -575,10 +589,21 @@ function discoverController($scope, config, courier, $route, $window, Notifier,
     columnActions.moveColumn($scope.state.columns, columnName, newIndex);
   };
 
-  $scope.toTop = function () {
+  $scope.scrollToTop = function () {
     $window.scrollTo(0, 0);
   };
 
+  $scope.scrollToBottom = function () {
+    // delay scrolling to after the rows have been rendered
+    $timeout(() => {
+      $document.find('#discoverBottomMarker').focus();
```

Comment:
No, it's just a remnant of the fact that the code lived deeper in the directive hierarchy previously. good catch